graphql-dgs-client/src/main/kotlin/com/netflix/graphql/dgs/client/codegen/GraphQLQuery.kt

```diff
@@ -18,9 +18,10 @@ package com.netflix.graphql.dgs.client.codegen
 
 import java.util.*
 
-abstract class GraphQLQuery(val operation: String) {
```

Comment:
We need the additional constructor here that takes in the name.

Reply:
Do you mean something like the following?

```kotlin
constructor(name: String) : this("query", name)
```

That would have the same signature as the existing following one, and unfortunately, it would not compile :(

```kotlin
constructor(operation: String) : this(operation, null)
```

Reply:
Sorry, what I mean for that is 
```
constructor(operation: String, name: String) : this("query", name)
``` 
in addition to 
```constructor(operation: String) : this(operation, null)
```

Reply:
Hi @srinivasankavitha, I'm a bit confused.

Isn't that the primary constructor already? https://github.com/Netflix/dgs-framework/pull/195/files/5dde55c5a57c884e45ed961060dc3edac3fb63a8#diff-c4fd37e186aa99535318e2b88b6058255c651239802ca9a1df7cd181b857d835R21

Thanks!

Reply:
I missed that, sorry for the confusion (apparently viewing code on the phone is not ideal :)). That should take care of backwards compatibility in that case.